integration-cli/docker_cli_build_test.go

```diff
@@ -1964,7 +1964,7 @@ func TestBuildForceRm(t *testing.T) {
 // * Wait for docker events to emit a dying event.
 func TestBuildCancelationKillsSleep(t *testing.T) {
 	// TODO(jfrazelle): Make this work on Windows.
-	testRequires(t, SameHostDaemon)
+	//testRequires(t, SameHostDaemon)
```

Comment:
@duglin yes I just pushed to see what's happening on Windows, I'm not sure it works btw